oc_cdtapi/tests/test_JenkinsAPI.py

```
@@ -197,7 +198,7 @@ def get( self, str_url, *args, **kwargs ):
 
             if str_job_action == "config.xml" :
                 if not str_job_name in self.dict_jobs_exist:
-                    jerr = JenkinsError( str_job_name + " not found" );
+                    jerr = HttpAPIError( str_job_name + " not found" );
```

Ruff (0.15.7)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)